apps/foundation-demo/src/app/design/molecules/footer/components/footer/footer.component.scss

```diff
@@ -0,0 +1,33 @@
+@import 'daffodil-util';
+
+.daff-footer {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr) 0.5fr;
+  grid-template-rows: auto;
+  grid-row-gap: 20px;
+  margin: 35px 15px;
+
+  @include breakpoint(medium) {
+    margin: 0 50px 100px 50px;
+    grid-template-columns: repeat(2, 1fr) repeat(2, 0.75fr);
+  }
+
+  @include breakpoint(large) {
+    margin: 0 150px 100px 200px;
+    grid-template-columns: repeat(4, 1fr);
+  }
+
+  &__icons {
```

Comment:
I needed it to be really generic, so that a user can just pass it icons without worrying about styling the icons themselves. Also, I needed to make the content "float right" without actually using float, so I used the "align-items: end" instead. I basically just made an icon container that looks ok no matter how many icons you give it, and I think I would need to micro-manage icons more with a horizontal list setup.

Reply:
@lderrickable agree with @damienwebdev on the `button-list`.

Reply:
Will `button-list` be a separate PR?

Reply:
I guess I don't know what a "button-list" is. Just a list of buttons?

Reply:
@xelaint yes.